src/routes/transactions/mappers/common/custom-transaction.mapper.spec.ts

```diff
@@ -127,13 +148,13 @@ describe('Multisig Custom Transaction mapper (Unit)', () => {
     const toAddress = new AddressInfo(faker.finance.ethereumAddress());
     addressInfoHelper.getOrDefault.mockResolvedValue(toAddress);
     const method = faker.word.sample();
-    const value = faker.number.int();
+    const value = '0';
```

Comment:
Was this changed to avoid big values with faker?

Reply:
I think it was wrong. See line 157 below pre-change. A cancellation transaction [has a value of '0'](https://github.com/safe-global/safe-client-gateway-nest/blob/scientific-notation/src/routes/transactions/mappers/common/custom-transaction.mapper.ts#L71).

Reply:
Oh I see. Yeah this is about cancellation transactions so it makes sense 👍